docs/variants.rst

```diff
@@ -8,8 +8,23 @@ and combine a common set of build options and give them a useful name. The main
 interface for creating and building variants is ``cmake-variants.json``, or
 ``cmake-variants.yaml``.
 
+The idea of "variants" are separate from the concept of toolchains/toolsets,
+which are handled via :ref:`Kits <kits>`.
+
+By default, if no variants file is present, CMake Tools will load four variants,
+each corresponding to a default CMake build type: *Release*, *Debug*,
+*MinSizeRel*, and *RelWithDebInfo*. Selecting one of these variants will
+configure and build with that build type.
+
+.. note::
+    CMake Tools does not presently respect ``CMAKE_CONFIGURATION_TYPES``. Only
+    the default four will be present. A custom variant file is required to load
+    these build types.
+
 For smaller or simple projects, providing a custom ``cmake-variants.yaml`` is
-unnecessary, and the default CMake build types will work just fine.
+unnecessary, and the default CMake build types will work just fine. Large
+projects with more complex configuration options make want to specify
```

Comment:
`complex configuration options make want to specify` -> `complex configuration options may want to specify`

Reply:
still relevant